DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -210,6 +210,14 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
 
         // pipe additional parameters unchanged to jackalope
         $backendParameters += $session['backend']['parameters'];
+        if (array_key_exists('curl_options', $session['backend']) && count($session['backend']['curl_options'])) {
+            $curlOptions = array();
+            foreach ($session['backend']['curl_options'] as $option => $value) {
+                $curlOptions[constant('CURLOPT_'.$option)] = $value;
```

Comment:
i would prefer to have them write the full CURLOPT_XX thing and require it to be uppercase. the less variation and magic, the easier to understand.

and lets check is_int to allow people using php configuration to use the constants themselves.